packages/@aws-cdk/aws-sqs/test/test.sqs.ts

```diff
@@ -276,21 +278,44 @@ export = {
     },
   },
 
+  'test ".fifo" suffixed queues register as fifo'(test: Test) {
+    const stack = new Stack();
+    const queue = new Queue(stack, 'Queue', {
+      queueName: 'MyQueue.fifo'
```

Comment:
Also verify the behavior when queueName is not specified but `fifo: true` is specified.